interview-questions.md

```diff
@@ -7,60 +7,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is a version control system created by Linus Torvalds to assist in the development of the Linux OS. It's taken over as the industry standard from an older system called Subversion. Git's main advantage is that it's decentralized, meaning it doesn't need a single server for people to work off of. It's decentralized nature is why GitHub exists. Anyone can push thier changes up to GitHub and allow anyone else to contribute to thier projects. GitHub is the cloud service that runs Git.
 
-  Researched answer:
+  Researched answer: Basically the same thing I wrote above. Git is the software and GitHub is the cloud service.
```

Comment:
I can appreciate the good work you did. Your researched answer is an opportunity to dive deeper on the subject or even tangential subjects. How does the cloud service work? Who made it?  Is it related to git? etc...

Reply:
Got it. I'll expand on it more thoroughly next assignment. Thx!